paper-slider.html

```diff
@@ -452,7 +452,9 @@
         markers: {
           type: Array,
           readOnly: true,
-          value: []
+          value: function () {
```

Comment:
nit: no space between function keyword and brackets for anonymous functions `function() {`